src/renderer/pages/ConnectForm.tsx

```diff
@@ -180,6 +235,39 @@ const ConnectForm: FC<Props> = () => {
     }
   };
 
+  const [clientCertFiltersExpanded, setClientCertFiltersExpanded] =
+    useState(false);
+
+  const saveClientCertFromStore = (
+    value: ClientCertFromStore | undefined,
+  ): void => {
+    setConnection({
+      ...connection,
+      ...{ clientCertFromStore: value },
+    });
+    if (value === undefined) {
```

Comment:
Does this work now. Usually I see 
typeof value === 'undefined' 
or just
 !value

Reply:
Good question... according to https://stackoverflow.com/questions/4725603/variable-undefined-vs-typeof-variable-undefined I think it should be fine.

If I understand correctly the `typeof` check is useful when an identifier might not have been declared. Here `value` is a function parameter so we know it's declared.

I want to compare against `undefined` explicitly because in the API proto there's a distinction between `clientCertFromStore` being unset (meaning doesn't search the OS certificate store) vs. being set but empty (meaning search the OS certificate store, without any additional filters).

Reply:
Makes sense